src/DacpacTool/Program.cs

```diff
@@ -33,6 +33,16 @@ static async Task<int> Main(string[] args)
             };
             buildCommand.Handler = CommandHandler.Create<BuildOptions>(BuildDacpac);
 
+            var inspectCommand = new Command("inspect")
```

Comment:
This is starting to become a little bit messy now. We might want to move to a model where each command is implement in its own class. I'll try to come up with a design for that, let's keep it like it is for now though.

Reply:
Agreed, but not making changes as part of this PR, per your comment.